packages/core/test/unit/configuration.test.ts

```diff
@@ -250,82 +166,5 @@ describe('Sequelize constructor', () => {
       });
       expect(replication.read).to.deep.eq([]);
     });
-
-    it('accepts the postgresql protocol', () => {
-      // should not throw
-      new Sequelize({
-        dialect: PostgresDialect,
-        url: `postgresql://localhost:9821/dbname?ssl=true`,
-      });
-    });
-  });
-
-  describe('SQLite URL parsing', () => {
```

Comment:
Do these already come back in the form of sqlite specific tests for storage?

Reply:
Added one for both temporary databases